package.json

```diff
@@ -27,7 +27,8 @@
     "@probe.gl/bench": "^4.0.0",
     "@turf/destination": "^6.0.1",
     "@types/tape-promise": "^4.0.1",
-    "ocular-dev-tools": "2.0.0-alpha.34",
+    "@vis.gl/dev-tools": "1.0.0-alpha.7",
+    "@vis.gl/ts-plugins": "1.0.0-alpha.7",
```

Comment:
Is there a usability advantage to requiring apps to manage two separate imports? Or could dev-tools depend on/re-export ts-plugins directly?